cf-admin.php

```diff
@@ -297,4 +301,4 @@ static function activate_plugin_for_new_blog($file, $blog_id, $single_activation
 		}
 	}
 
-} // end class-exists check ?>
\ No newline at end of file
+} // end class-exists check ?>
```

Comment:
Comment should start with uppercase and end with a dot.
Also, you can remove the ending PHP tag `?>`, if it's at the end of the file.